.gitignore

```diff
@@ -0,0 +1 @@
+.idea/*
```

Comment:
what's this for?

Reply:
Just for opening it with goland..

Reply:
not clear though, why the codebase keep something for a specific IDE?

Reply:
Goland is widely used in go projects, for go code or some docs.. like K8s: https://github.com/kubernetes/kubernetes/blob/master/.gitignore#L13, 
 Istio https://github.com/istio/istio/blob/master/.gitignore#L10